channels/sessions.py

```diff
@@ -183,7 +182,7 @@ async def send(self, message):
                 and message.get("status", 200) != 500
                 and (modified or settings.SESSION_SAVE_EVERY_REQUEST)
             ):
-                await database_sync_to_async(self.save_session)()
```

Comment:
Removal of this is a small breaking change right? 

I think it's probably OK, but we should call it out strongly.

Reply:
Documented in the release notes!